<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Contour ShuttleXpress</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#main_widget {background: url(images/ShuttleXpress_Black.png);
     background-repeat:no-repeat;}
    </string>
   </property>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>197</x>
      <y>178</y>
      <width>216</width>
      <height>216</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black;</string>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>9</number>
    </property>
    <property name="sliderPosition">
     <number>2</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>true</bool>
    </property>
    <property name="notchTarget">
     <double>3.700000000000000</double>
    </property>
    <property name="notchesVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos1">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>151</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg3">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>211</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos7">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>378</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg4">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>253</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg6">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>341</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg5">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>296</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg1">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>151</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos2">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>174</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos6">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>341</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="button_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>266</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>253</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos5">
    <property name="geometry">
     <rect>
      <x>419</x>
      <y>296</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg2">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>174</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_cent0">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>143</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_pos3">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>211</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="wheel_neg7">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>378</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="button_2">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>122</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="button_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>122</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="button_5">
    <property name="geometry">
     <rect>
      <x>498</x>
      <y>266</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="button_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>72</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #000000ff;
      color: white;
     </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="about_button">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>0</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="about_widget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>580</width>
     <height>574</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="about_widget_contents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QTextEdit" name="about_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>550</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <zorder>about_widget</zorder>
 </widget>
 <tabstops>
  <tabstop>button_1</tabstop>
  <tabstop>button_2</tabstop>
  <tabstop>button_3</tabstop>
  <tabstop>button_4</tabstop>
  <tabstop>button_5</tabstop>
  <tabstop>wheel_neg7</tabstop>
  <tabstop>wheel_neg6</tabstop>
  <tabstop>wheel_neg5</tabstop>
  <tabstop>wheel_neg4</tabstop>
  <tabstop>wheel_neg3</tabstop>
  <tabstop>wheel_neg2</tabstop>
  <tabstop>wheel_neg1</tabstop>
  <tabstop>wheel_cent0</tabstop>
  <tabstop>wheel_pos1</tabstop>
  <tabstop>wheel_pos2</tabstop>
  <tabstop>wheel_pos3</tabstop>
  <tabstop>wheel_pos4</tabstop>
  <tabstop>wheel_pos5</tabstop>
  <tabstop>wheel_pos6</tabstop>
  <tabstop>wheel_pos7</tabstop>
  <tabstop>dial</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
